README.md

```diff
@@ -35,7 +35,22 @@ https://docs.microsoft.com/cs-cz/dotnet/core/install/macos
 
 ## Produkční deployment
 
-TBD
```

Comment:
Nerozjebe to teď leteronův PR #4? Se tam bude míchat čeština a angličtina.

Reply:
Ano. Rozbije. Aby se to vyřešilo, tak bude nutné prvně sloučit #4 a poté upravit rozdíly v tomto PR.